docs/community/topics/i18n.rst

```diff
@@ -92,14 +104,36 @@ This section covers how to do so.
 
       pybabel init -i src/pydata_sphinx_theme/locale/sphinx.pot -d src/pydata_sphinx_theme/locale -D sphinx -l es
 
+   **To run this in ``.nox``**: ``nox -s translate -- init es``
+
 #. Edit the language's message catalog at ``pydata_sphinx_theme/locale/es/LC_MESSAGES/sphinx.po``. For each source string introduced by the ``msgid`` keyword, add its translation after the ``msgstr`` keyword.
 
 #. Compile the message catalogs of every language. This creates or updates the MO files with `PyBabel compile <https://babel.pocoo.org/en/latest/cmdline.html#compile>`__:
 
-.. code-block:: bash
+   .. code-block:: bash
+
+      pybabel compile -d src/pydata_sphinx_theme/theme/pydata_sphinx_theme/static/locale -D sphinx
+
+   **To run this in ``.nox``**: ``nox -s translate -- compile``.
+
+Translation tips
+----------------
+
+Translating words vs phrases
+````````````````````````````
+
+Full sentences and clauses must always be a single translatable string.
+Otherwise, you get ``next page`` translated as ``suivant page`` instead of ``page suivante`` etc.
+
+Dealing with names and non-translateable text in translations
+`````````````````````````````````````````````````````````````
 
-   pybabel compile -d src/pydata_sphinx_theme/locale -D sphinx
+In some cases we must include non-translateable text within translations (e.g. ``GitHub``).
+It's important to include them in the translatable phrase so that word ordering is accounted for in the translations.
+To do so, use Jinja syntax to insert the name as a variable (``{{ variablename }}``).
```

Comment:
```suggestion
If a variable (like the ``edit_page_provider_name`` theme option) is used as part of a phrase, it must be included within the translatable string.
Otherwise, the word order in other languages might be incorrect.
In a Jinja template, simply surround the translatable string with ``{% trans variable=variable %}`` and ``{% endtrans %}}`.
For example: ``{% trans provider=provider %}Edit on {{ provider }}{% endtrans %}``
The translatable string is extracted as the Python format string ``Edit on %(provider)s``.
This is so that the same translatable string can be used in both Python code and Jinja templates.
It is the translator's responsibility to use ``%(provider)s`` verbatim in the translation.

If a non-translatable word or token (like HTML markup) is used as part of a phrase, it must also be included within the translatable string.
For example: ``{% trans theme_version=theme_version|e %}Built with the <a href="https://pydata-sphinx-theme.readthedocs.io/en/stable/index.html">PyData Sphinx Theme</a> {{ theme_version }}.{% endtrans %}``
It is the translator's responsibility to use the HTML markup verbatim in the translation.
```